tonic_validate/metrics/metric.py

```diff
@@ -11,6 +12,12 @@ def name(self) -> str:
         """Metric name for the UI"""
         pass
 
+    @property
```

Comment:
We don't actually need `@property` here. We only did `@property` for name since we let people name some metrics themselves when they instantiate. Here the prompt will remain constant so we don't need to do property